From ltsi-dev-bounces@lists.linuxfoundation.org Wed Nov  5 10:40:01 2014
From: Chang Rebecca Swee Fun <rebecca.swee.fun.chang@intel.com>
Date: Wed,  5 Nov 2014 09:39:13 +0800
Subject: [LTSI-dev]  [PATCH 08/16] mfd: lpc_ich: Change Avoton to iTCO v3
To: LTSI Mailing List <ltsi-dev@lists.linuxfoundation.org>
Cc: Chang Rebecca Swee Fun <rebecca.swee.fun.chang@intel.com>
Message-ID: <1415151561-16047-9-git-send-email-rebecca.swee.fun.chang@intel.com>


From: Peter Tyser <ptyser@xes-inc.com>

The register layout of the Avoton is compatible with the iTCO v3
register layout.

Signed-off-by: Peter Tyser <ptyser@xes-inc.com>
Tested-by: Rajat Jain <rajatjain@juniper.net>
Reviewed-by: Guenter Roeck <linux@roeck-us.net>
Signed-off-by: Lee Jones <lee.jones@linaro.org>
(cherry picked from commit c48cf59878685cc06b71bb2a3ca17b61103c8de7)

Signed-off-by: Chang Rebecca Swee Fun <rebecca.swee.fun.chang@intel.com>
---
 drivers/mfd/lpc_ich.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/mfd/lpc_ich.c
+++ b/drivers/mfd/lpc_ich.c
@@ -501,7 +501,7 @@ static struct lpc_ich_info lpc_chipset_i
 	},
 	[LPC_AVN] = {
 		.name = "Avoton SoC",
-		.iTCO_version = 1,
+		.iTCO_version = 3,
 		.gpio_version = AVOTON_GPIO,
 	},
 	[LPC_COLETO] = {
